packages/demo/components/pages/apis/transaction/builder/mintNative.tsx

```diff
@@ -5,7 +5,7 @@ export default function Mint() {
   return (
     <Section
       sidebarTo="mintNative"
-      header="Mint native script token"
+      header="Mint tokens with a 'native script'"
```

Comment:
Not sure if this change is good. Also it bump the text box size on nav menu on the left. I would not prefer having this change, or maybe you can supply more reason behind this favoured change

Reply:
Purpose was to make it clearer what the purpose is, for those less familiar with the various types, but can keep it how you prefer.